app/javascript/pages/getAssistance/document-checklist.tsx

```diff
@@ -27,39 +30,39 @@ const DocumentChecklist = () => {
       ),
     },
     {
-      title: t("e2cLiveWorkPreference.liveSfPreference.title"),
+      id: PREFERENCES_IDS.liveWorkInSf,
+      title: t("listings.lotteryPreference.Live or Work in San Francisco Preference.title"),
       expanded: (
-        <div className="space-y-2.5">
-          <p>{t("documentChecklist.liveSfDoc1")}</p>
-          <p>{t("documentChecklist.liveSfDoc2")}</p>
-          <ul className="list-disc ml-7">
-            <li>{t("label.proof.telephoneBill")}</li>
-            <li>{t("label.proof.cableBill")}</li>
-            <li>{t("label.proof.electricBill")}</li>
-            <li>{t("label.proof.gasBill")}</li>
-            <li>{t("label.proof.waterBill")}</li>
-            <li>{t("label.proof.paystubHome")}</li>
-            <li>{t("label.proof.publicBenefits")}</li>
-            <li>{t("label.proof.schoolRecord")}</li>
-            <li>{t("label.proof.homelessness")}</li>
-          </ul>
-        </div>
-      ),
-    },
-    {
-      title: t("e2cLiveWorkPreference.workSfPreference.title"),
-      expanded: (
-        <div className="space-y-2.5">
-          <p>{t("documentChecklist.workSfDoc1")}</p>
-          <p>{t("documentChecklist.workSfDoc2")}</p>
-          <ul className="list-disc ml-7">
-            <li>{t("label.proof.paystubEmployer")}</li>
-            <li>{t("label.proof.letterFromEmployer")}</li>
+        <div>
+          <p className="pb-2.5">{t("documentChecklist.twoWaysDesc")}</p>
+          <ul className="space-y-2.5 list-decimal list-inside ml-2">
+            <li>
```

Comment:
I am not sure why, but it looks like the "If you already live in San Francisco. As proof, you will need to show" text is being put below the `<li>`
<img width="1080" alt="Screenshot 2023-05-23 at 9 22 56 PM" src="https://github.com/SFDigitalServices/sf-dahlia-web/assets/131277283/c80f6b56-ca45-46b6-818d-4a43395c5675">